torchmetrics/text/rouge.py

```diff
@@ -76,6 +86,7 @@ class ROUGEScore(Metric):
     def __init__(
         self,
         use_stemmer: bool = False,
+        accumulate: str = "best",
```

Comment:
```suggestion
        accumulate: Literal["avg", "best"] = "best",
```

and `from typing_extensions import Literal`